salt/modules/openscap.py

```diff
@@ -107,3 +107,7 @@ def xccdf(params):
         upload_dir=upload_dir,
         error=error,
         returncode=returncode)
+
+
+def xccdf_eval(profile=None, policy=None):
+    return xccdf('eval --profile {0} {1}'.format(profile, policy))
```

Comment:
I am very confused why we even need all this. Why `openscap.xccdf` cannot behave like `openscal.xccdf_eval` and the latter can cease to exist?